msg/follow_target_status.msg

```diff
@@ -0,0 +1,18 @@
+uint64 timestamp                  # [microseconds] time since system start
+
+float32[3] pos_est_filtered       # [m/s] Filtered target position used for vehicle's setpoint generation
+float32[3] vel_est_filtered       # [m/s] Filtered Target velocity used for vehicle's setpoint generation
+
+float32 tracked_target_course     # [rad] Tracked target course in NED local frame (North is course zero)
+float32 follow_angle              # [rad] Current follow angle setting
+
+float32 orbit_angle_setpoint      # [rad] Current orbit angle setpoint from the smooth trajectory generator
+float32 raw_orbit_angle_setpoint  # [rad] Raw 'idealistic' orbit angle setpoint (has discrete jumps)
+
+bool in_emergency_ascent          # [bool] True when doing emergency ascent (when distance to ground is below safety altitude)
+float32 gimbal_pitch              # [rad] Gimbal pitch commanded to track target in the center of the frame
+
+float32 max_angular_rate_setpoint # [rad/s] Maximum angular rate setpoint calculated from remaining angle & jerk, accel settings
+float32 angular_rate_setpoint     # [rad/s] Angular rate commanded from Jerk-limited Orbit Angle trajectory for Orbit Angle
+
+float32[3] desired_position_raw   # [m] Raw 'idealistic' desired drone position if a drone could teleport from place to places
```

Comment:
Those are 34 more fields to log which adds flash for every field name. For comparison that's the same amount as the entire `estimator_status` message.

Reply:
Right. I think a lot of those were necessary for debugging purposes, but can be removed now.

Reply:
I think we can remove
- pos_est_filtered
- vel_est_filtered
- raw_orbit_angle_setpoint
- desired_position_raw

@junwoo091400 what's `tracked_target_course`? Is that the estimated target course?

Reply:
Yes it's the estimated target course!

Reply:
Some of the inner fields were removed & was incorporated into the commit!

I left the desired_position_raw in there as it gives a intuitive sense on what the 'correct' follow position is, but I am also up for removing it :wink: